src/strategies/samurailegends-generals-balance/index.ts

```diff
@@ -0,0 +1,43 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { Multicaller } from '../../utils';
+
+export const author = 'Samurai-Legends';
+export const version = '0.1.2';
+
+const abi = [
+  'function erc721GetAllTokensOfOwner(address nftAddress, address user) external view returns (uint[] memory)'
+];
```

Comment:
Hey @ThijmenMaus Why not use https://snapshot.org/#/strategy/erc721 strategy? 🤔

Reply:
Is it possible to apply that together with the limitations of max id 5000?

Reply:
@ChaituVR

Reply:
Oh i see it now, i read the Readme and thought this is just getting balance of users

Reply:
Thanks for the information, I will remember to add information regarding that in the readme next time.